src/main/java/com/fakng/fakngagrgtr/parser/apple/ResponseDTO.java

```diff
@@ -0,0 +1,37 @@
+package com.fakng.fakngagrgtr.parser.apple;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties({
```

Comment:
Is it required to list these properties explicitly? I expected that ObjectMapper just checks the ones, that are defined in the class and ignores all others
Am I wrong?

Reply:
Maybe there is some kind of setting (I'll look at it later), but if you don't specify it, the error "found a field that is not specified as jsonIgnored" will pop up. Something like this.

Reply:
Okay :( 
That's sad

Yep, please, google it up. If there's no way to do it prettier, we'll leave it like this for now

Reply:
@JsonIgnoreProperties(ignoreUnknown = true)

Here it is, I found it!

Reply:
It's great, thanks :)

Reply:
I found a better option. Set the property to ObjectMapper itself.